db/schema.rb

```diff
@@ -10,74 +10,8 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema[7.0].define(version: 2023_09_19_064605) do
+ActiveRecord::Schema[7.0].define(version: 0) do
```

Comment:
Any issues with the migrations!

Reply:
not directly. But when I'm changing the branch, the tables still exist on the DB. So when i want to migrate down in the other branch I always have to delete the tables, which do not exist there...

Reply:
You can try using Github Desktop to manage pushing necessary files.